components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.spec.ts

```diff
@@ -28,60 +31,9 @@ describe('PendingEditsBarComponent', () => {
   let store: Store<NgrxStateAtom>;
 
   beforeEach(async(() => {
-
-    TestBed.configureTestingModule({
-      declarations: [
-        MockComponent({
-          selector: 'chef-toolbar',
-          template: '<ng-content></ng-content>'
-        }),
-         MockComponent({
-          selector: 'app-authorized',
-          inputs: ['allOf', 'not'],
-          template: '<ng-content></ng-content>'
-        }),
-        MockComponent({
-          selector: 'app-message-modal',
-          inputs: ['visible' ],
-          outputs: ['close' ]
-        }),
-         MockComponent({
-          selector: 'app-confirm-apply-start-modal',
-          inputs: ['visible'],
-          outputs: ['confirm', 'cancel']
-        }),
-        MockComponent({
-          selector: 'app-confirm-apply-stop-modal',
-          inputs: ['visible', 'applyRulesStatus', 'stopRulesInProgress'],
-          outputs: ['confirm', 'cancel']
-        }),
-        MockComponent({ selector: 'mat-progress-bar', inputs: ['mode', 'value', 'bufferValue'] }),
-        MockComponent({ selector: 'mat-select' }),
-        MockComponent({ selector: 'mat-option' }),
-        MockComponent({ selector: 'chef-button', inputs: ['disabled'] }),
-        MockComponent({ selector: 'chef-heading' }),
-        MockComponent({ selector: 'chef-loading-spinner' }),
-        MockComponent({ selector: 'chef-page-header' }),
-        MockComponent({ selector: 'chef-subheading' }),
-        MockComponent({ selector: 'chef-table-new' }),
-        MockComponent({ selector: 'chef-table-header' }),
-        MockComponent({ selector: 'chef-table-body' }),
-        MockComponent({ selector: 'chef-table-row' }),
-        MockComponent({ selector: 'chef-table-header-cell' }),
-        MockComponent({ selector: 'chef-table-cell' }),
-        PendingEditsBarComponent
-      ],
-      imports: [
-        ReactiveFormsModule,
-        RouterTestingModule,
-        ChefPipesModule,
-        StoreModule.forRoot(ngrxReducers, { runtimeChecks })
-      ],
-      providers: [
-        FeatureFlagsService,
-        ProjectService
-      ]
-    }).compileComponents();
+      configureWith({
+        '/iam/v2/projects': genPerm('/iam/v2/projects', true)
+      });
```

Comment:
Needed to be able to apply different configurations in `TestBed.configureTestingModule` so moved it to a helper function. Notice that in this call, we are showing the user has permission to do a `GetProjects` call. This is the data used by introspection to report that.